packages/calcite-components/src/components/input/input.scss

```diff
@@ -301,6 +384,8 @@ input[type="time"]::-webkit-clear-button {
   @apply pointer-events-none
     absolute
     block;
+  --calcite-progress-background-color: var(--calcite-input-loading-background-color);
```

Comment:
![Screenshot 2024-11-13 at 12 58 37 PM](https://github.com/user-attachments/assets/5b59e913-4756-4d4a-8f45-c7ab2ddaf482)

The current set up has a bit of an overlap issue with the progress bar, if the corner radius is set on the Input. We might be able to work around this by moving the progress bar's location within the component and handling with overflow css, but haven't looked yet.

Reply:
This means the Progress Bar would get clipped by the Input, correct? Long term I think we discussed entertaining the Loader icon instead of the Progress Bar to denote a loading Input which would nicely avoid this concern.